pagopa/io-app-design-system
# [IOAPPX-320] Completely replace `TitilliumWeb` with `TitilliumSansPro`

## Short description
This PR completely replaces `TitilliumWeb` with `TitilliumSansPro` (designed and developed by [Chialab](https://github.com/chialab/titillium_pro))

## List of changes proposed in this pull request
- Remove `TitilliumWeb` references from the components and codebase
- Remove `TitilliumWeb` from platform related files (`*.pbxproj`, etc…)
- Update `TitilliumSansPro` font files with latest developments

## How to test
1. Launch the example app
2. Check if every component is okay
